css/styles.css

```diff
@@ -1129,15 +1129,14 @@ a.plugin_formcreator_formTile_title {
    clear: both;
    position: fixed;
    bottom: 0;
-   width:  100%;
-   padding: 5px 0;
+   width:  calc(100% - 10px);
```

Comment:
I don't really like this solution as it depends on other values:
```css
#footer {
   font-size: 9px;
   color: #000;
   background-color: #E2E1DE;
   padding: 6px;
   height: 18px;
   padding: 5px;  <-----------------------------------

   > table {
      width: 100%;
   }
}
```

If someone change this padding, the calc will not work as expected as it need 100% - (padding-left + padding right).

If you really want to do it this way, maybe use CSS vars ?